sql/core/src/main/scala/org/apache/spark/sql/SQLContext.scala

```diff
@@ -328,18 +328,19 @@ class SQLContext(@transient val sparkContext: SparkContext)
     def numPartitions = self.numShufflePartitions
 
     def strategies: Seq[Strategy] =
-      extraStrategies ++ (
```

Comment:
Seems a bug of error reporting, anyway, the root reason is it failed in the code style checking.
`/home/jenkins/workspace/SparkPullRequestBuilder/sql/core/src/main/scala/org/apache/spark/sql/execution/commands.scala:198:21: Insert a space after the start of the comment`